src/fpylll/algorithms/bkz.py

```diff
@@ -85,7 +85,10 @@ def __call__(self, params, min_row=0, max_row=-1):
 
         i = 0
         while True:
-            with tracer.context("tour", i):
+            with tracer.context(
+                    "tour",
+                    i,
+                    dump_gso=params.flags & BKZ.DUMP_GSO):
```

Comment:
I'm not a big fan of these additional linebreaks, to me, it makes it harder to not easier to read

Reply:
Agreed, I just wasn't sure how strict to be about flake8 compliance.